package.json

```diff
@@ -229,7 +231,8 @@
 		"playground:dev": "concurrently \"npm run dev:packages\" \"parcel playground/src/index.html -d playground/dist\"",
 		"preenv": "npm run check-engines",
 		"env": "wp-scripts env",
-		"design-system:dev": "concurrently \"npm run dev:packages\" \"start-storybook -c ./packages/components/storybook\"",
+		"design-system:dev": "concurrently \"npm run dev:packages\" \"./node_modules/@storybook/react/bin/index.js -c ./packages/components/storybook\"",
+		"design-system:dev:native": "concurrently \"npm run dev:packages\" \"(adb reverse tcp:7007 tcp:7007 || true) && ./node_modules/@storybook/react-native-server/bin/index.js -c ./packages/components/storybook\"",
```

Comment:
This is pretty strange that they use the same name for binaries:

https://github.com/storybookjs/storybook/blob/4bd4888de89c3439b9bc2a39e725884ca25dfe4a/app/react-native-server/package.json#L24-L27

🙃

Reply:
I double-checked, those are 2 different servers:

https://unpkg.com/@storybook/react@5.2.3/dist/server/index.js
https://unpkg.com/browse/@storybook/react-native-server@5.2.3/dist/server/index.js